packages/postgrest/lib/src/postgrest_query_builder.dart

```diff
@@ -280,4 +249,15 @@ class PostgrestQueryBuilder<T> extends PostgrestBuilder<T, T> {
     }
     return _url;
   }
+
+  /// Only performs a count query on the table or view.
+  /// ```dart
+  /// int count = await supabase.from('users').count();
+  /// ```
+  RawPostgrestBuilder<int, int, int> count(CountOption option) {
```

Comment:
I'm up for discussion, but I wonder if it makes sense to have a default value here. 
```suggestion
  RawPostgrestBuilder<int, int, int> count([CountOption option = CountOption.exact]) {
```

Reply:
I like the idea!